{"timestamp":"2025-07-07T09:02:31.381855","level":"info","event":"DAG bundles loaded: dags-folder, example_dags","logger":"airflow.dag_processing.bundles.manager.DagBundlesManager"}
{"timestamp":"2025-07-07T09:02:31.382562","level":"info","event":"Filling up the DagBag from /opt/airflow/dags/polars_example.py","logger":"airflow.models.dagbag.DagBag"}
{"timestamp":"2025-07-07T09:02:31.573283Z","level":"info","event":"Task instance is in running state","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-07T09:02:31.573762Z","level":"info","event":" Previous state of the Task instance: TaskInstanceState.QUEUED","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-07T09:02:31.574366Z","level":"info","event":"Current task name:transform","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-07T09:02:31.574610Z","level":"info","event":"Dag name:polars_etl_pipeline","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-07T09:02:31.659546","level":"error","event":"Task failed with exception","logger":"task","error_detail":[{"exc_type":"AttributeError","exc_value":"'DataFrame' object has no attribute 'groupby'","exc_notes":[],"syntax_error":null,"is_cause":false,"frames":[{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":867,"name":"run"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/task_runner.py","lineno":1159,"name":"_execute_task"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/bases/operator.py","lineno":397,"name":"wrapper"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":216,"name":"execute"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/providers/standard/operators/python.py","lineno":239,"name":"execute_callable"},{"filename":"/home/airflow/.local/lib/python3.12/site-packages/airflow/sdk/execution_time/callback_runner.py","lineno":81,"name":"run"},{"filename":"/opt/airflow/dags/polars_example.py","lineno":22,"name":"transform_orders"}],"is_group":false,"exceptions":[]}]}
{"timestamp":"2025-07-07T09:02:31.660769Z","level":"info","event":"Task instance in failure state","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-07T09:02:31.661282Z","level":"info","event":"Task start","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-07T09:02:31.661522Z","level":"info","event":"Task:<Task(PythonOperator): transform>","chan":"stdout","logger":"task"}
{"timestamp":"2025-07-07T09:02:31.661761Z","level":"info","event":"Failure caused by 'DataFrame' object has no attribute 'groupby'","chan":"stdout","logger":"task"}
